openedx/openedx-platform
# Auto-advance video toggle

Replaced by #15509.

## Description

This adds an auto-advance toggle to the video player.  If turned on, as soon as the video ends the next unit or subsection will be loaded.

Since this is a global preference, when it is set loading a section with a video will automatically start it.  This has the (desired) effect that consecutive video sections will play automatically from start to finish.

## Dependencies

n/a

## JIRA

n/a

## Sandbox URLs

- [LMS](https://pr15034.sandbox.opencraft.hosting/)
- [Studio](https://studio-pr15034.sandbox.opencraft.hosting/)

## Latest commit

- f5bae43 *(updated 2017-05-18)*

## Testing instructions

1. Go to the [pre-deployed course](https://pr15034.sandbox.opencraft.hosting/courses/course-v1:OpenCraft+DEMO+DEMO/about), and log in as `staff@example.com`.
2. Got to the first section, and enable the "Auto-advance" button on the video.  (It should turn blue.)
3. Wait for the video to end.  The next unit should load automatically, and the video should start playing with no user input.

## Screenshots

### Disabled
![screenshot](https://cloud.githubusercontent.com/assets/759355/26214766/fa99a1ae-3bd3-11e7-83b4-9a3619d87906.png)

### Enabled
![screenshot from 2017-05-18 14-10-54](https://cloud.githubusercontent.com/assets/759355/26214781/05349cae-3bd4-11e7-8b21-372b0de9ca4e.png)
